src/server/lib/__tests__/okta/register.test.ts

```diff
@@ -108,14 +109,19 @@ jest.mock('@/server/lib/trackMetric', () => ({
 }));
 
 const email = 'someemail';
-const User = (status: Status, hasPassword = true): UserResponse => {
+const User = (
+  status: Status,
+  hasPassword = true,
+  registrationLocation: RegistrationLocation | undefined = undefined,
+): UserResponse => {
   return {
     id: 'someuserid',
     status: status,
     profile: {
       login: email,
       isGuardianUser: true,
       email: email,
+      registrationLocation: registrationLocation,
```

Comment:
Is this neccessary? not being used for anything other than the server unit test.

Reply:
I suppose if `registrationLocation` can end up in a user response it might be good to keep it in the mocked response in case of unexcpected side-effects?

Reply:
Sorry, linked to the wrong file. I added it to the Okta Response in the **User Model** purely to add a server unit test. It is  not being used in gateway no shown back to the user, nor does it need to be returned in the user response. 

However, we may want to keep in the UserResponse model purely so that the response reflects data that has been updated in Okta? I have left it in for this reason.